packages/tools/devtools/devtools-view/src/components/index.ts

```diff
@@ -8,6 +8,7 @@
 /* eslint-disable no-restricted-syntax */
 
 export * from "./data-visualization";
+export * from "./graphs";
```

Comment:
We probably don't need to export from `components/` yet, I think? The views where we'd want to use charts are inside this module and they can `import {<stuff>} from "./graphs";`, right?